src/app/icd/ICDManagementServer.h

```diff
@@ -53,6 +53,8 @@ class ICDManagementServer
     Status UnregisterClient(PersistentStorageDelegate & storage, FabricIndex fabric_index, chip::NodeId node_id,
                             Optional<chip::ByteSpan> verificationKey, bool is_admin);
 
+    void TableIsEmptyForFabric();
```

Comment:
This needs to be documented. Why does it take no arguments for fabric, or return nothing?